aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1475,6 +1475,71 @@ module aptos_token::token {
         token_data.royalty
     }
 
+    /// return the token_data_id from the token_id
```

Comment:
Can we separate these getter functions out into a different PR? These seem generally useful.

Reply:
done